testfm/packages.py

```diff
@@ -13,6 +13,7 @@
     install                       Install packages in an unlocked session
     update                        Update packages in an unlocked session
     is-locked                     Check if update of packages is allowed
+    check-update                  Check if update of packages is available
```

Comment:
Here help description seems different, seems to be entered manually, we should be following exact details and order for metadatas.

Reply:
Okay